### ISSUE TITLE
questions about code

### ISSUE THREAD

**Issue by mulinmeng** [AutoVision-cloud/Deformable-PV-RCNN#8]
https://github.com/AutoVision-cloud/Deformable-PV-RCNN/blob/9833efa8bb2e9e873ce1bb3df1cb3ed368cb6232/src/pointnet_modules/pointnet2_modules.py#L262

Set here as 'torch.no_grad()', will the self.pred_offset.weight not be learnt?

**Comment by AutoVision-cloud** [comment]
Technically, the query and group part does not need to pass gradients, but when passed specifically through the stack implementation for PCDet, this threw an error (from my understanding this was not the case for batch implementation but a second opinion is welcome). This was the easiest work-around for this situation.
